include/ralf_pd_definitions.h

```diff
@@ -124,4 +124,29 @@ static const PDLog2<const char*, int> CL_RALF_DIAMETER_STOP_FAIL
   "No action required."
 );
 
+static const PDLog1<const char*> CL_RALF_CDF_CONNECTION_ERROR
+(
+  PDLogBase::CL_RALF_ID + 10,
+  LOG_INFO,
+  "Connections between Ralf and CDF applications at %s have failed.",
+  "At least one of the Diameter peers is not contactable and the number of contactable "
+  "instances is strictly less than expected (two unless changed through configuration)",
+  "This process was unable to contact some instances of the application "
```

Comment:
This can be more specific! The reason for having separate PDLogs is so we can avoid these generic sentences that don't help with debugging. I've got the same comment for the Homestead PD logs.